### PR TITLE
ENH: Clean up docker file

### LINKED ISSUE #6: Redundancy
https://github.com/scil-vital/dockerfile_tractolearn/blob/main/Dockerfile#L57

The directory change in there is redundant as there is another one changing to the same directory immediately before.

### PR DESCRIPTION
**Pull request by jhlegarreta**
- STYLE: Remove trailing whitespace and add newline at EOF
- ENH: Remove redundant directory change statement

Resolves #6.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -41,15 +41,14 @@ RUN rm -rf /var/lib/apt/lists/* && \
         | tar -xz -C /opt/ants-2.3.4 --strip-components 1
 
 RUN pip install -U pip
```
Comment (jhlegarreta) [inline Dockerfile]: Is this necessary?

File: Dockerfile [hunk 2]
```diff
@@ -41,15 +41,14 @@ RUN rm -rf /var/lib/apt/lists/* && \
...
-RUN cd /tractolearn && pip install -e . 
+RUN cd /tractolearn && pip install -e .
```
Comment (jhlegarreta) [inline Dockerfile:43]: We are using here a bare `pip` whereas all other commands use the Python 3.8 version `pip`. I would use `pip` everywhere to save typing and make maintenance easier. As we are installing a single Python version, `pip` should be able to resolve to that verstion.

Comment (felixdumais) [reply Dockerfile:43]: Yep, I tested it on my side, you can remove the pip3.8

File: Dockerfile (hunk 1 above)
Comment (felixdumais) [reply Dockerfile]: Nope, you can remove

File: Dockerfile (hunk 2 above)
Comment (jhlegarreta) [reply Dockerfile:43]: Done in [268d578](https://github.com/scil-vital/dockerfile_tractolearn/pull/7/commits/268d578fa57f899c826f542f9055a68583682063).

File: Dockerfile (hunk 1 above)
Comment (jhlegarreta) [reply Dockerfile]: Done in [6888d13](https://github.com/scil-vital/dockerfile_tractolearn/pull/7/commits/6888d13dd817844f74983c5d0115dd5075b4e1f2).

**Pull request merged**